Library/ExternalSubscription.cs

```diff
@@ -0,0 +1,189 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Xml;
+
+namespace Recurly
+{
+
+    /// <summary>
+    /// Represents external_subscriptions for accounts
+    /// </summary>
+    public class ExternalSubscription : RecurlyEntity
+    {
+        private string _accountCode;
+        public string AccountCode => _accountCode;
+
+        private Account _account;
+        public Account Account
+        {
+            get { return _account ?? (_account = Accounts.Get(_accountCode)); }
+            internal set { _account = value; }
+        }
+
+        public string ExternalObjectReference { get; private set; }
+        public string Uuid { get; private set; }
+        public string AppIdentifier { get; private set; }
+        public bool? AutoRenew { get; set; }
+        public int Quantity { get; set; }
+        public ExternalProductReference ExternalProductReference { get; private set; }
+        public DateTime? LastPurchased { get; private set; }
+        public DateTime? ActivatedAt { get; private set; }
+        public DateTime? ExpiresAt { get; private set; }
+        public DateTime CreatedAt { get; private set; }
+        public DateTime UpdatedAt { get; private set; }
```

Comment:
If top level nodes are `last_purchased, auto_renew, app_identifier, quantity, activated_at expires_at, created_at, updated_at` can we access Uuid Here?If yes, do you know where it's coming from?

Reply:
Meant to delete that.